project_template/{{module_name}}/__main__.py

```diff
@@ -0,0 +1,8 @@
+from calculator import Calculator
+calc = Calculator()
+
+calc.add(5)
+
+calc.subtract(2)
+
+print(calc.cumulative_total)
```

Comment:
This would currently raise lint failures.
```suggestion
from calculator import Calculator

calc = Calculator()
calc.add(5)
calc.subtract(2)

print(calc.cumulative_total)

```